database_manager.py

```diff
@@ -1,29 +1,58 @@
 import database_helper
 
+
 class ItemManager:
 
     def __init__(self, database_connector):
         self.database_connector = database_connector
-    
-    def add_item(self):
-        item_id = database_helper.get_uuid()
-        self.database_connector.execute("INSERT INTO Items (ItemId, VectorPath, PngPath) VALUES (%s, %s, %s)", (item_id, item_id+"_vector", item_id+"_png"))
-        return item_id
+
+    def add_item(self, item_id):
+        self.database_connector.execute(
+            "INSERT INTO items (ItemId, VectorPath, PngPath) VALUES (%s, %s, %s)",
+            (item_id, item_id+"_vector", item_id+"_png"))
 
     def delete_item(self, item_id):
-        self.database_connector.execute("DELETE FROM Items WHERE ItemId=%s", (item_id))
-    
+        self.database_connector.execute(
+            "DELETE FROM Items WHERE ItemId=%s", item_id)
+
+    def get_item_details(self, item_id):
+        item_details = self.database_connector.execute(
+            "SELECT * from Items WHERE ItemId=%s", item_id)
+        assert len(item_details) == 1
+        return item_details[0]
```

Comment:
A suggestion (for a different comment perhaps) is to create a NamedTuple as a type holding items, and return that instead. Returning a dict with keys can be a bit unclear, and also looks messy.